### ISSUE TITLE
Create strange dataType when using with database migration

### ISSUE THREAD

**Issue by confile** [robertoschwald/grails-audit-logging-plugin#106]
When I create a database migration I get the following changeset entries: 

```
	changeSet(author: "mg (generated)", id: "1444221549231-2") {
		modifyDataType(columnName: "new_value", newDataType: "varchar(65534)", tableName: "audit_log")
	}

	changeSet(author: "mg (generated)", id: "1444221549231-3") {
		modifyDataType(columnName: "old_value", newDataType: "varchar(65534)", tableName: "audit_log")
	}
```

I use MySql and always have to change the datatype from ``varchar(65534)`` to ``longtext``.

**Comment by robertoschwald** [comment]
What Grails and Hibernate version?

**Comment by confile** [comment]
Grails 2.4.5

runtime ":hibernate:3.6.10.18"

runtime 'mysql:mysql-connector-java:5.1.36'

**Comment by robertoschwald** [comment]
Which version of audit-logging?

**Comment by confile** [comment]
compile ":audit-logging:1.0.5"

**Comment by robertoschwald** [comment]
Strange. In my projects (mostly Grails 2.4.4, Hibernate 3 and 4, db-migration 1.4.0) I never had this issue and in all projects I get:

```groovy
	column(name: "new_value_da", type: "longtext")
	column(name: "old_value_da", type: "longtext")
```

Normally, Grails should pick the right column type for Strings based on constraint size. This doesn't seem to work in your case.
If you are using Intellij, can you please test the following:

 - Open the audit-logging plugin source in your IDEA project (Grails view, plugins > audit-logging-1.0.5) and modify the AuditLogEvent class: 

 - add to the mapping closure:
```groovy
static mapping = {
 ...
 oldValue  type: "text"
}
```

Then generate a new db-migration and check.

You must perform grails clean after the test to get the original plugin code back.

**Comment by robertoschwald** [comment]
Verified here in a new project. Its an issue. Will provide a fix for the 1.0.x series.

1.1.0-SNAPSHOT2 is not affected, as in this upcoming 1.x release, users generate the AuditLogEvent domain artifact in their project.

**Comment by robertoschwald** [comment]
Please use fixed version 1.0.7.
